src/odemis/util/test/comp_test.py

```diff
@@ -235,6 +235,18 @@ def test_clipping_zmax_on_actuator_upper_limit(self):
         actual = generate_zlevels(self.focus, zrange, zStep)
         numpy.testing.assert_array_almost_equal(expected, actual)
 
+    def test_clipping_axis_range(self):
+        self.focus.updateMetadata({model.MD_POS_ACTIVE_RANGE: {"z": [-100e-6, 100e-6]}})
+        try:
+            self.focus.moveAbsSync({"z": 80e-6})
+            zrange = (-50e-6, 50e-6)
+            zStep = 10e-6
```

Comment:
I would suggest to keep the variable name consistent with `zrange`
```suggestion
            zstep = 10e-6
```